FYDP_ModV1/src/version1.java

```diff
@@ -306,14 +410,14 @@ public static void modelConfig() throws IOException {
 		}*/
 				
 	//time periods array
```

Comment:
we can gas this- i set it up like that for reading it in but you did that anyway so we can chuck it

Reply:
commented section removed!